### DISCUSSION TITLE
Dynatrace Metrics

### DISCUSSION THREAD

**Discussion by ghost** [Dynatrace/dynatrace-configuration-as-code#197] [Archive]
Should monaco have an integration option whereby it pushes metrics into an environment whenever it performs an action?

ie. 3 configs downloaded or 27 configs uploaded / amended

**Comment by tatomir146** [comment]
Monaco should only be used for pushing configuration, not metrics. Closing this discussion.

**Reply by ghost** [reply]
I don't mean that. I mean that whenever monaco is used, it pushes a metric into Dynatrace to tell people that it was used
